common/meta/types/src/user_auth.rs

```diff
@@ -163,7 +171,7 @@ impl AuthInfo {
                 hash_value: p,
                 hash_method: t,
             } => t.to_string(p),
-            AuthInfo::None => "".to_string(),
+            AuthInfo::None | AuthInfo::JWT => "".to_string(),
```

Comment:
typo? it seems jwt has already pattern matched in an earlier match arm.

Reply:
@flaneur2020  in two diff functions get_type and get_auth_string

Reply:
oh my bad, sorry :joy: